base/errorshow.jl

```diff
@@ -698,7 +691,7 @@ end
 function print_stackframe(io, i, frame::StackFrame, n::Int, digit_align_width, modulecolor)
     file, line = string(frame.file), frame.line
     stacktrace_expand_basepaths() && (file = something(find_source_file(file), file))
-    stacktrace_contract_userdir() && (file = replaceuserpath(file))
+    stacktrace_contract_userdir() && (file = contractuser(file))
```

Comment:
This doesn't seem to work on Windows (which might be why the custom function was called).

Reply:
That is exactly the wrong reason to define a new function however

Reply:
Can you open a new issue to change that function? It is okay to assign me.